src/util/defaults.ml

```diff
@@ -134,6 +134,7 @@ let _ = ()
       ; reg Analyses "ana.sv-comp.functions" "false" "Handle SV-COMP __VERIFIER* functions"
       ; reg Analyses "ana.specification"   "" "SV-COMP specification (path or string)"
       ; reg Analyses "ana.wp"              "false" "Weakest precondition feasibility analysis for SV-COMP violations"
+      ; reg Analyses "ana.nullptr"        "false"  "Null Pointer dereferencing check"
```

Comment:
Is it really necessary to have this option here? The warning options already control whether to show this (among unknown pointer dereferencing warnings, which don't have such corresponding option) and it's just one extra check.

Reply:
sure I can remove the flag!

Reply:
removed the flag! :)